traitsui/testing/tester/wx/helpers.py

```diff
@@ -76,6 +76,39 @@ def mouse_click_object(control, delay):
     control.ProcessEvent(click_event)
 
 
+def mouse_click_notebook_tab_index(control, index, delay=0):
+    """ Performs a mouseclick on a Noteboook List Editor on the tab specified
+    by index.
+
```

Comment:
This function could probably be used in other scenarios beyond just for the Notebook editor, or at least modified to do so.  For now though, its only use case is the Notebook editor.